lib/ansible/modules/cloud/amazon/route53.py

```diff
@@ -178,7 +178,7 @@
       type: bool
       sample: false
     failover:
-      description:
+      description: ""
```

Comment:
did you intend to include this file in the PR?

Reply:
Yeah... I could do a separate PR but... it's just making the docs build work because those empty description lines were translating to None.

Reply:
For clarity... None leads to a Not Iterable traceback when building docs.

Reply:
Okay, just wanted to make sure!  I wasn't opposed to it being included.  Just wanted to raise a flag just incase 😄.